code/modules/power/cell.dm

```diff
@@ -118,9 +118,8 @@
 		rigged = 1 //broken batterys are dangerous
 
 /obj/item/weapon/cell/emp_act(severity)
-	charge -= 1000 / severity
-	if (charge < 0)
-		charge = 0
+	var/powerloss = round(16 * sqrt(maxcharge) / severity, 50) //at severity 1, ~500 for 1000 power cells, ~2750 for 30,000 power cells
```

Comment:
At most effective severity (1)
Drains about 50% of a 1000 power cell, 
Drains about 20% of a 7500 power cell,
Drains about 10% of a 30,000 power cell.